docs/content/users/extend/custom-commands.md

```diff
@@ -218,6 +218,14 @@ If your container command should run from the directory you are running the comm
 
 Example: `## HostWorkingDir: true`
 
+### “ExecRaw” Annotation (Container Commands Only)
+
+Use `ExecRaw: true` to pass arguments for the command you want to run directly to the command in the container as-is. 
```

Comment:
This sentence seems a little tricky to parse. Maybe this whittles it down without losing meaning?

```suggestion
Use `ExecRaw: true` to pass command arguments directly to the container as-is. 
```